ts/sagas/profile.ts

```diff
@@ -63,8 +63,47 @@ import {
   getLocalePrimaryWithFallback
 } from "../utils/locale";
 import { readablePrivacyReport } from "../utils/reporters";
+import { LollipopConfig } from "./../features/lollipop";
 
 // A saga to load the Profile.
+export function* loadProfileLollipop(
+  lollipopConfig: LollipopConfig,
+  getProfileLollipop: ReturnType<typeof BackendClient>["getProfileLollipop"]
+): Generator<
+  ReduxSagaEffect,
+  O.Option<InitializedProfile>,
+  SagaCallReturnType<typeof getProfileLollipop>
+> {
+  try {
+    const response = yield* call(getProfileLollipop, lollipopConfig);
+    // we got an error, throw it
+    if (E.isLeft(response)) {
+      throw Error(readablePrivacyReport(response.left));
+    }
+    if (response.right.status === 200) {
+      // Ok we got a valid response, send a SESSION_LOAD_SUCCESS action
+      // BEWARE: we need to cast to UserProfileUnion to make UserProfile a
```

Comment:
you mean` to InitializedProfile`?

Reply:
This is a legacy comment. Anyway `loadProfileLollipop` has been deleted because it was only intended to be there during development.